src/netbase.cpp

```diff
@@ -692,7 +694,6 @@ bool SetSocketNonBlocking(SOCKET& hSocket, bool fNonBlocking)
         int fFlags = fcntl(hSocket, F_GETFL, 0);
```

Comment:
I guess the argument can be just `SOCKET `, or even `const SOCKET` now that it's no longer changed?